konami/identifiers.md

```diff
@@ -29,7 +30,7 @@ The following variables are used across the different identifiers and can denote
   * `K`: Possibility for offline/no e-AMUSEMENT?
   * `N`: New dedicated cabinet?
   * `O`: New dedicated cabinet?
-  * `S`: ???
+  * `S`: Station (medal games)
```

Comment:
While I agree that this is a likely explanation, I've seen dongles that explicitly append this information, e.g. `GSPTL-ST` and `RTL-JB[STATION]` -- it would seem like a weird redundancy. Also, would it mean new station cabinet specifically, or something else?

Reply:
Hmm, it might just be for medal games in general then. 
``GSPTLJBA`` and ``GSRTLJBA`` are the security codes on PTL-002/RTL-002 "station" dongles, at least. (cert and bootstrap.xml)

Reply:
I'll change it to `Medal cabinet (station?)` to indicate the uncertainty. I wonder if the server cabinets (in as much as they exist) have different codes.

Reply:
They have their own contents codes, at least (RTL-001/PTL-001). I assume anything that has a -002 also has a -001.
In MFC/QMA's case, it's just -JA vs -JB. (older QMA might have been CE/CL judging by security codes). QMA has almost always shared data and dongles (despite different labels, LMA for example has -JA and -JB and possibly -JD for the new HG cabinets, despite all being LMA-001), but MFC has always had everything different (IK9-001/IK9-002, KK9-001/KK9-002)